src/Template/Element/top-authenticated.ctp

```diff
@@ -25,6 +26,19 @@
 		<!-- Collect the nav links, forms, and other content for toggling -->
 		<div class="collapse navbar-collapse navbar-ex1-collapse">
 			<ul class="nav navbar-nav navbar-right">
+
+				<!-- notifications -->
+				<?php if (empty($data['notifications'])) : ?>
+					<li class="navbar-notifier">
+						<a href="#">Messages <span class="badge">0</span></a>
+					</li>
+				<?php else : ?>
+					<li class="navbar-notifier has-messages">
+						<a data-toggle="collapse" href="#collapseNotifier" aria-expanded="false" aria-controls="collapseExample">Messages <span class="badge"><?= count($data['notifications']) ?></span></a>
```

Comment:
refs my earlier question about nested array, there will always be only one notification or none.